src/test/java/uk/ac/ebi/ampt2d/accession/HashMapTestAccessionRepository.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.accession;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class HashMapTestAccessionRepository implements AccessionRepository<String> {
```

Comment:
The public interface operates with Map (not HashMap), so `TestMapAccessionRepository` could be a better name. Note the "Test" before "Map" for consistency with `TestAccessionGeneratorA`.